### PR TITLE
testing/perl-crypt-openssl-x509: new aport

### PR DESCRIPTION
**Pull request by timlegge**

### CODE REVIEW

**Comment by Ikke** [comment]
This counts for all the perl related pull requests you've opened: Could you please remove the empty variables?

Thanks.

File: testing/perl-crypt-openssl-x509/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Maintainer: Timothy Legge <timlegge@gmail.com>
+pkgname=perl-crypt-openssl-x509
+_pkgreal=Crypt-OpenSSL-X509
+pkgver=1.812
+pkgrel=0
+pkgdesc="Perl extension to OpenSSL's X509 API."
+url="https://metacpan.org/release/Crypt-OpenSSL-X509/"
+arch="all"
+license="GPL PerlArtistic"
+cpandepends=""
```
Comment (Ikke) [inline testing/perl-crypt-openssl-x509/APKBUILD]: Please remove empty global variables.

File: testing/perl-crypt-openssl-x509/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+cpancheckdepends=""
+depends="$cpandepends"
+makedepends="perl-dev $cpanmakedepends"
+checkdepends="$cpancheckdepends"
+subpackages="$pkgname-doc"
+source="https://cpan.metacpan.org/authors/id/J/JO/JONASBN/Crypt-OpenSSL-X509-$pkgver.tar.gz"
+builddir="$srcdir/$_pkgreal-$pkgver"
+
+build() {
+	cd "$builddir"
```
Comment (Ikke) [inline testing/perl-crypt-openssl-x509/APKBUILD]: These can be removed as well, as abuild is now taking care of it.

Comment (timlegge) [reply testing/perl-crypt-openssl-x509/APKBUILD]: Done, will address the others.  I seem to have remembered that part way through the PRs

**Comment by timlegge** [comment]
I pushed a force before I noticed this one.  Were you saying the that cd
$builddir can be removed?

Tim

**Comment by timlegge** [comment]
As in:

`diff --git a/testing/perl-moosex/APKBUILD b/testing/perl-moosex/APKBUILD
index f254d97706..8620e6ec19 100644
--- a/testing/perl-moosex/APKBUILD
+++ b/testing/perl-moosex/APKBUILD
@@ -17,20 +17,17 @@ source="https://cpan.metacpan.org/authors/id/E/ET/ETHER/MooseX-$pkgver.tar.gz"
 builddir="$srcdir/$_pkgreal-$pkgver"
 
 build() {
-       cd "$builddir"
        export CFLAGS=$(perl -MConfig -E 'say $Config{ccflags}')
        PERL_MM_USE_DEFAULT=1 perl -I. Makefile.PL INSTALLDIRS=vendor
        make
 }
 
 check() {
-       cd "$builddir"
        export CFLAGS=$(perl -MConfig -E 'say $Config{ccflags}')
        make test
 }
 
 package() {
-       cd "$builddir"
        make DESTDIR="$pkgdir" install
        find "$pkgdir" \( -name perllocal.pod -o -name .packlist \) -delete
 }
`
If so I will update and submit a PR for abuild/apkbuild-cpan.in

**Pull request closed without merging**